addons/web/static/src/js/fields/basic_fields.js

```diff
@@ -2275,26 +2282,13 @@ var FieldDomain = AbstractField.extend({
  */
 var AceEditor = DebouncedField.extend({
     template: "AceEditor",
+    jsLibs: [[
+        '/web/static/lib/ace/ace.odoo-custom.js',
+        '/web/static/lib/ace/mode-python.js',
+        '/web/static/lib/ace/mode-xml.js',
+    ]],
```

Comment:
[[ace.odoo-custom], [mode-python, mode-xml]]

Reply:
ok but it's equal to [ace.odoo-custom, [mode-python, mode-xml]]